app/controllers/conference_controller.rb

```diff
@@ -5,7 +5,7 @@ class ConferenceController < ApplicationController
   load_resource :program, through: :conference, singleton: true, except: :index
 
   def index
-    @current = Conference.where('end_date >= ?', Date.current).order('start_date ASC')
+    @current = Conference.where('end_date >= ?', Date.current).reorder('start_date ASC')
```

Comment:
Why DESC? Don't we want to show the earliest conference first? (ie. first show oSC16 and then show summit asia in events.o.o)